src/transformStateWithClones.js

```diff
@@ -8,6 +8,32 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const newObj = { ...state };
+  const newArr = [];
+
+  for (const i of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
i хорошее название для итератора в цикле, но плохое, когда там хранится объект